index.html

```diff
@@ -26,7 +26,7 @@ <h1>Added some stuff</h1>
     </div>
     <div class="row">
       <div class="col-md-3">
-        <h1>Ryeker</h1>
+        <h1>Vinni</h1>
       </div>
       <div class="col-md-9">
         <h3>I heard it on the radio, wanted a new job. etc.</h3>
```

Comment:
You still need to change this line. You can make the changes in the same branch. Just add, commit and push and it will update this pull request so no need to create another.